src/for_serde.rs

```diff
@@ -0,0 +1,170 @@
+use std::io::{self, Write};
+use std::iter::Enumerate;
+use std::collections::btree_map::Iter as MapIter;
+use std::slice::Iter as VecIter;
+
+use serde_json::value::Value;
+
+use super::ToGron;
+
+enum StackItem<'a> {
+    MapIter(MapIter<'a, String, Value>),
+    VecIter(Enumerate<VecIter<'a, Value>>),
+}
+
+
+impl ToGron for Value {
+    /// Converts JSON structure from `serde::value::Value` into gron format
```

Comment:
Docstring in trait implementation is useless I think (does it even show up in docs?). Move (and edit) comment to original `json_to_gron` function